### PR TITLE
chore: adjust time for stale github issues to close after 1 week of inactivity

### PR DESCRIPTION
**Pull request by AgustinBettati**
## Description

Ticket: [INTMDB-1181](https://jira.mongodb.org/browse/INTMDB-1181)

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -12,11 +12,11 @@ jobs:
...
+          stale-issue-message: 'This issue has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 7 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
+          stale-pr-message: 'This PR has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 7 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
           stale-issue-label: 'stale'
           stale-pr-label: 'stale'
           days-before-stale: 30
-          days-before-close: 30
```
Comment (andreaangiolillo) [inline .github/workflows/stale.yml:20]: Do we want to update `days-before-stale: 30` as well or rely completely to us adding the stale flag manually on the issue?

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -12,11 +12,11 @@ jobs:
...
+          days-before-close: 7
           exempt-pr-labels: 'not_stale'
           exempt-issue-labels: 'not_stale'
```
Comment (andreaangiolillo) [inline .github/workflows/stale.yml:22]: You also need to update the readme file https://github.com/mongodb/terraform-provider-mongodbatlas#autoclose-stale-issues-and-prs

File: .github/workflows/stale.yml (hunk 1 above)
Comment (AgustinBettati) [reply .github/workflows/stale.yml:20]: Had this doubt, giving it some thought I think it is better to automatically mark as stale after 1 week and if issue is relevant we will adjust to not_stale

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: .github/workflows/stale.yml (hunk 1 above)
Comment (andreaangiolillo) [reply .github/workflows/stale.yml:20]: Maybe it makes more sense to update them to 5 days instead of 7

**Pull request merged**